test/libraries/SafeMigration.spec.ts

```diff
@@ -46,6 +46,14 @@ const migrationPaths = [
 ];
 
 describe("SafeMigration Library", () => {
+    before(function () {
+        /**
+         * ## Migration tests are not working yet for zkSync: this test depends on the EVM bytecode
+         * which is not supported on zkSync. Tests will be adjusted.
```

Comment:
Should we create a GH issue to track this?

Reply:
I think it's covered by https://github.com/safe-global/safe-smart-account/issues/767